README.md

```diff
@@ -359,13 +359,13 @@ The most common attacks are **white-box attacks**:
 
 It is developed in Python and implements extraction, inversion, poisoning and evasion attacks and defenses.
 
-ART supports the most popular frameworks: Tensorflow, Keras, PyTorch, MxNet, ScikitLearn, among many others.
+ART supports the most popular frameworks: Tensorflow, Keras, PyTorch, MxNet, and ScikitLearn among many others.
 
-It is not limited to the use of models that use images as input, but also supports other types of data, such as audio, video, tabular data, etc.
+It is not limited to the use of models that use images as input but also supports other types of data, such as audio, video, tabular data, etc.
 
 > [Workshop to learn Adversarial Machine Learning with ART 🇪🇸](https://github.com/jiep/adversarial-machine-learning)
 
-###### Cleverhans
+###### Clever Hans
```

Comment:
Please, update title to CleverHans. See official name: https://github.com/cleverhans-lab/cleverhans

Reply:
Can you review it now, and tell me if more changes are required, I am ready to help.